src/test/java/reactor/kafka/receiver/KafkaReceiverTest.java

```diff
@@ -245,7 +245,7 @@ public void manualAssignmentWithCommit() throws Exception {
         Flux<? extends ConsumerRecord<Integer, String>> kafkaFlux =
                 KafkaReceiver.create(receiverOptions)
                         .receive()
-                        .doOnNext(r -> r.receiverOffset().commit().block())
+                        .concatMap(r -> r.receiverOffset().commit().then(Mono.just(r)))
```

Comment:
why not just ``` .concatMap(r -> r.receiverOffset().commit()```

Reply:
Because if we do that we will lose `r` because in `doOnNext` the stream does not map one to another. But concateMap will map `ConsumerRecords` to `Void` so we will lose element.

Reply:
then delayUntil might be the right operation 💃